docs/en/integrations/language-clients/python/index.md

```diff
@@ -0,0 +1,316 @@
+                                                                                                                                                                                                                                                                                   |
+#### Additional Parameters
+
+Additional keyword args to `clickhouse_connect.get_client` not listed above are used as query parameters for all requests to the ClickHouse server.
+Because they are sent as query parameters, all values for these additional arguments are converted to strings.  
+The following parameters are related to the actual query or command:
+
+| Parameter         | Description                                                                                                                                                           |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| buffer_size       | Buffer size (in bytes) used by ClickHouse Server before writing to the HTTP channel.                                                                                  |
+| session_id        | A unique session id to associate related queries on the server.  Required for temporary tables.                                                                       |
+| compress          | Whether the ClickHouse server should compress the POST response data.  This setting should only be used for "raw" requests.                                           |
+| decompress        | Whether the data sent to ClickHouse server must be decompressed.  This setting is should only be used for "raw" requests.                                             |
+| quota_key         | The quota key associated with this requests.  See the ClickHouse server documentation on quotas.                                                                      |
+| session_check     | Used to check the session status.                                                                                                                                     |
+| session_timeout   | Number of seconds of inactivity before the identified by the session id will timeout and no longer be considered valid.  Defaults to 60 seconds.                      |
+| wait_end_of_query | Buffers the entire response on the ClickHouse server.  This setting is necessary to return summary information.  It is set automatically when *send_progress*=*True*. |
+
+All other keyword args and interpreted as ClickHouse user settings for each request. Please see the full ClickHouse  documentation for a complete list.  Note that
```

Comment:
Should we link https://clickhouse.com/docs/en/operations/settings/settings? There are many other settings available